drivers/video/video.c

```diff
@@ -3410,7 +3416,16 @@ int video_initialize(FAR const char *devpath)
 
 int video_uninitialize(FAR const char *devpath)
 {
-  return video_unregister(devpath);
```

Comment:
should we fix video_unregister?

Reply:
@xiaoxiang781216 
Sorry for the late reply.
I think "No".
For examples, drivers/video/goldfish.c use the video_register().
some drivers may use the video_register() and video_unregister().

Reply:
but if user call video_unregister instead video_uninitialize, the leak still happen too